src/components/MapSelectors/GlobalSelector.js

```diff
@@ -69,7 +69,7 @@ const CountyItem = ({ dataset }) => {
   );
 };
 
-const GlobalSelector = ({ handleChange, extendRight }) => {
+const GlobalSelector = ({ handleChange, extendRight, onIsFocusedChanged }) => {
```

Comment:
nit: handleChange / onIsFocusedChanged isn't very consistent naming.  And you named it "focused" but it's actually driven by the `isOpen` state.  Maybe `handleIsOpen`?

Minor and maybe we don't care at this point. Feel free to punt.

Reply:
`handleIsOpen` is clear and consistent! I'll use it.